alacritty_terminal/src/term/mod.rs

```diff
@@ -422,11 +423,11 @@ impl<'a, C> Iterator for RenderableCellsIter<'a, C> {
                     let mut cell = RenderableCell::new(self, cell);
                     cell.inner = RenderableCellContent::Cursor(self.cursor.key);
 
-                    // Only apply static color if it isn't close to the cell's current background.
-                    if let CellRgb::Rgb(color) = self.cursor.cursor_color {
-                        if color.contrast(cell.bg) >= MIN_CURSOR_CONTRAST {
-                            cell.fg = self.cursor.cursor_color.color(cell.fg, cell.bg);
-                        }
+                    // Only apply static color if it isn't close to the cell's current
+                    // background or refers to cell colors.
```

Comment:
The latter part is already implied by `static`.

Maybe something like this is more clear:

```suggestion
                    // Apply cursor color, or invert the cursor if it has a fixed background close to the cell's background.
```

I'm not sure which of these I like more. The one describes the purpose of this block better, but describes something that isn't done, while the other isn't entirely clear